x/txfees/keeper/hooks.go

```diff
@@ -224,6 +224,8 @@ func (k Keeper) calculateDistributeAndTrackTakerFees(ctx sdk.Context, defaultFee
 func (k Keeper) swapNonNativeFeeToDenom(ctx sdk.Context, denomToSwapTo string, feeCollectorAddress sdk.AccAddress) sdk.Coin {
 	coinsToSwap := k.bankKeeper.GetAllBalances(ctx, feeCollectorAddress)
 	totalCoinOut := sdk.NewCoin(denomToSwapTo, osmomath.ZeroInt())
+	coinsNotSwapped := []string{}
```

Comment:
We don't do anything with this variable? Presumably were supposed to log it if its not empty?

Reply:
I think it got removed when I merged in main :/

Reply:
added back https://github.com/osmosis-labs/osmosis/pull/7660/commits/efaa89efe751206a4f059f2dc268c79775da7f68